### PR TITLE
Keyboard Shortcut

### PR DESCRIPTION
**Pull request by antv11a**
Adds a keyboard shortcut for the slider.

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -5,5 +5,14 @@
     "version": "0.0.1",
     "action": {
       "default_popup": "popup/popup.html"
+    },
+    "permissions": ["activeTab", "scripting"],
```
Comment (mvilla9) [inline manifest.json:9]: Why do we need to add these permissions?

**Review by mvilla9 — COMMENTED** [review]
Looks like you used the documentation to add some keyboard shortcut functionality, which is good. There's a lot of stuff here that we do not need though and `_execute_action` is not how we should implement this. This issue was marked as an enhancement, so it is not a priority right now. Let's get basic functionality implemented before we start working on shortcuts

File: manifest.json (hunk 1 above)
Comment (antv11a) [reply manifest.json:9]: Honestly, I just read the google developer article and did what they did. But got it, I'll start working on the functionality stuff that you assigned me.

Comment (mvilla9) [reply manifest.json:9]: When people are installing the extension, they may get alerts that they need to approve access to search history, webpage content, etc. if we have permissions defined that we don't actually need. We will need some permissions but we will add them as we work on tasks

**Pull request closed without merging**